Tests/Spec/tentacle_scenario_01_install_spec.rb

```diff
@@ -67,6 +67,7 @@
   it { should have_role('Test-Tentacle') }
   it { should have_display_name("#{ENV['COMPUTERNAME']}_PollingTentacle") }
   it { should have_policy('Default Machine Policy') }
+  it { should be_in_space('AdditionalSpace') }
```

Comment:
**Please note** that this will not work until the (https://github.com/OctopusDeploy/octopus-serverspec-extensions) is extended to support such a test.

Reply:
https://github.com/OctopusDeploy/octopus-serverspec-extensions/pull/3